src/uiSelectChoicesDirective.js

```diff
@@ -5,6 +5,8 @@ uis.directive('uiSelectChoices',
   return {
     restrict: 'EA',
     require: '^uiSelect',
+    priority: 1,
```

Comment:
This is a bad hack - one should never have to do priority: 1 to work around issues.